### ISSUE TITLE
Nested array list with 5+ items isn't fully accessible.

### ISSUE THREAD

**Issue by fusepilot** [sanity-io/sanity#2549]
**Describe the bug**

The list of items you can add to an array is cut off when the array is part of an object. Making some of the items inaccessible/unusable.

**To Reproduce**

1. Add array type to object with at least 6 `of` options.
2. Insert the object into portable text.
2. Click on '+ Add item...'
3. See that only the bottom 5 items are visible and preceding items are cut off.

**Expected behavior**

All of the items should be accessible instead of just the bottom 5.

**Screenshots**
![image](https://user-images.githubusercontent.com/187164/121596509-59291580-c9f4-11eb-8100-0f2273400f4d.png)

**Which versions of Sanity are you using?**

@sanity/cli                     2.10.5 (up to date)
@sanity/base                    2.10.5 (up to date)
@sanity/block-content-to-react   3.0.0 (up to date)
@sanity/cli                     2.10.5 (up to date)
@sanity/components               2.2.6 (up to date)
@sanity/core                    2.10.5 (up to date)
@sanity/default-layout          2.10.5 (up to date)
@sanity/default-login           2.10.5 (up to date)
@sanity/desk-tool               2.10.5 (up to date)
@sanity/production-preview       2.2.6 (up to date)
@sanity/vision                  2.10.5 (up to date)

**What operating system are you using?**

macOS 11.3.1
Chrome 91.0.4472.77

**Which versions of Node.js / npm are you running?**

6.14.12
v12.22.1

**Comment by fusepilot** [comment]
Adding this CSS override seems to fix it. Not sure what it will break elsewhere.

````
:global([class*="DefaultDialog_contentWithoutFooter_"]) {
  overflow-x: visible;
  overflow-y: visible;
}
````

**Comment by fabien** [comment]
I think a fix has been merged but It’s not released yet: https://github.com/sanity-io/sanity/pull/2528 - duplicate of https://github.com/sanity-io/sanity/issues/2499

**Comment by judofyr** [comment]
Closing as this seems to be fixed by #2528. Feel free to re-open if this is a mistake.
